src/app/diff-template/diff-template.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { diffJson } from 'diff'
+
+@Component({
+  selector: 'diff-template',
```

Comment:
don't call a component template, templates have different meaning in angular context, so `diff-view` would be better.